apps/condo/domains/user/schema/User.js

```diff
@@ -205,7 +207,15 @@ const User = new GQLListSchema('User', {
             }
         },
     },
-    plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
+    plugins: [
+        uuided(),
+        versioned(),
+        tracked(),
+        softDeleted(),
+        dvAndSender(),
+        historical(),
+        webHooked(webhookModelValidator),
```

Comment:
it's better to define `webhookModelValidator` inside the plugin logic

Reply:
Outdated, changed it in: https://github.com/open-condo-software/condo/pull/2497/commits/86e9c28e85f95fab65325596b6d4e74903d0d7c9